src/Tribe/Tickets_View.php

```diff
@@ -518,7 +526,19 @@ public function inject_link_template_the_content( $content ) {
 		}
 
 		ob_start();
-		include Tribe__Tickets__Templates::get_template_hierarchy( 'tickets/orders-link.php' );
+
+		$file = Tribe__Tickets__Templates::get_template_hierarchy( 'tickets/orders-link.php' );
+
+		/**
+		 * @since TBD Attempt to load from old location to account for pre-existing theme overrides. If not found,
+		 *            go through the motions with the new location.
+		 */
+		if ( empty( $file) ) {
```

Comment:
:no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 765557d9b1bb0b2ea45f99b7f5d341bc</sub>